gits/test/helper.py

```diff
@@ -347,33 +347,38 @@ def _check_cap_csr(self, reg):
         self.assertEqual(top - 1, term._top_most)
 
     @reset_after_executing
-    def _check_cap_dl1(self, s, pos):
-        """A helper that checks the `_cap_dl1` method.
-
-        The ``s`` argument is a test string to be put on the screen.
-        The ``pos`` argument must be a tuple or list of coordinates ``(x, y)``
-        of the location where you want the string to be.
+    def _check_cap_dl(self, n, lines_positions):
```

Comment:
Probably we should rename `lines_positions` into `lines` because the name doesn't reflect the purpose of the variable. It consists of both lines and its positions, but when I see `lines_positions` I have a strange feeling that it's a tuple (or something) of positions of lines.